### PR TITLE
Update documentation for BI tools

### LINKED ISSUE #72: [FEATURE] Enhance connection string documentation
We need to enhance the connection string documentation that we have available in the JDBC Driver. Using the driver with a BI tool should have better documentation.

### PR DESCRIPTION
**Pull request by simonz-bq**
### Summary

Update documentation for BI tools

### Description

* Fixed minor typo
* Added docs for DbVisualizer and DBeaver
* Updated README.md for more clarification

### Related Issue

https://bitquill.atlassian.net/browse/AN-871

### Additional Reviewers
@birschick-bq
@lyndonb-bq
@xiazcy
@simonz-bq
@alexey-temnikov
@kylepbit

### CODE REVIEW

File: markdown/bi-tools/DBeaver.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+### DBeaver
+
+Before proceeding, ensure you have [configured your environment](../setup/configuration.md).
+
+Download the latest version of [DBeaver](https://dbeaver.io/ to use the Driver in DBeaver.
```
Comment (alexey-temnikov) [inline markdown/bi-tools/DBeaver.md]: Missing `)`
```suggestion
Download the latest version of [DBeaver](https://dbeaver.io/) to use the Driver in DBeaver.
```

File: markdown/bi-tools/DBeaver.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+**For Mac you must launch your BI tool through terminal to have the environment variables loaded.**
```
Comment (alexey-temnikov) [inline markdown/bi-tools/DBeaver.md]: Need a reference or inline -  how to launch DBeaver from a command line and how to specify environment variable.

File: markdown/bi-tools/DbVisualizer.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+### DbVisualizer
+
+Before proceeding, ensure you have [configured your environment](../setup/configuration.md).
+
+Download the latest version of [DbVisualizer](https://www.dbvis.com/) to use the Driver in DbVisualizer.
+
+**For Mac you must launch your BI tool through terminal to have the environment variables loaded.**
```
Comment (alexey-temnikov) [inline markdown/bi-tools/DbVisualizer.md]: Description how to launch and specify variables (as in above)

File: README.md [hunk 4]
```diff
@@ -28,6 +28,10 @@ To set up a connection, the driver requires a JDBC connection URL. The connectio
 jdbc:neptune:[connectionType]://[host];[propertyKey1=value1];[propertyKey2=value2]..;[propertyKeyN=valueN]
 ```
 
+A basic example of a connection string is:
+
+```jdbc:neptune:sqlgremlin://neptune-example;port=8182```
```
Comment (lyndonbauto) [inline README.md]: I suggest we make this have `.com` at the end so it is more clear that the neptune-example part is a url, also it lines up with the example provided below.

```suggestion
```jdbc:neptune:sqlgremlin://neptune-example.com;port=8182```
```

File: markdown/bi-tools/DBeaver.md (hunk 2 above)
Comment (lyndonbauto) [reply markdown/bi-tools/DBeaver.md]: Do we care about environment variables loaded? We removed the dependency on this.

File: markdown/bi-tools/DbVisualizer.md (hunk 3 above)
Comment (lyndonbauto) [reply markdown/bi-tools/DbVisualizer.md]: Same comment, do we care about command line anymore? Could we just remove these statements entirely now?

File: markdown/bi-tools/tableau.md [hunk 5]
```diff
@@ -14,9 +12,10 @@ Download the latest version of [Tableau Desktop](https://www.tableau.com/product
 
 2. [Download](https://github.com/aws/amazon-neptune-jdbc-driver/releases) the Neptune Tableau connector (a `TACO` file) and copy it to your `My Tableau Repository/Connectors`
    directory.
+   
    - **_Windows_**: `C:\Users\[user]\Documents\My Tableau Repository\Connectors`
    - **_Mac_**: `/Users/[user]/Documents/My Tableau Repository/Connectors`
-
+
```
Comment (lyndonbauto) [inline markdown/bi-tools/tableau.md]:
```suggestion

```

**Review by lyndonbauto — APPROVED** [review]
LGTM, thanks sighmon

**Pull request merged**